### PR TITLE
CTCP-3186: CYA entry for transport equipment.

### PR DESCRIPTION
**Pull request by justega247**

### CODE REVIEW

File: test/services/TransportEquipmentsServiceSpec.scala [hunk 1]
```diff
@@ -77,5 +77,78 @@ class TransportEquipmentsServiceSpec extends SpecBase {
...
+            number = 0,
+            containerId = Some("98777")
+          )
+          result2.value mustBe TransportEquipment(
+            number = 1,
+            containerId = None
+          )
+        }
+      }
+
```
Comment (jackheslop96) [inline test/services/TransportEquipmentsServiceSpec.scala:144]: Can we also have a test where the json has an empty `equipments` array?

**Review by jackheslop96 — APPROVED** [review]

**Review by jackheslop96 — APPROVED** [review]
